src/app/components/Profile/Profile.module.css

```
@@ -0,0 +1,24 @@
+.profile {
+  color: white;
+  padding: 0.5rem;
+}
+
+.profile img {
+  border-radius: 50%;
+  margin-right: 1rem;
+}
+
+.profile--container {
+  display: flex;
+  align-items: center;
+}
+
+.profile--Error {
+  background-color: red;
+}
+
+.profile--login {
+  background-color: yellow;
+  padding: 0.5rem;
+  color: darkgray;
+}
```

We can reduce the complexity here because it's already targeting `profile` (no prefix required).

```suggestion
.container {
  color: white;
  padding: 0.5rem;
}

.avatar {
  border-radius: 50%;
  margin-right: 1rem;
}

.header{
  display: flex;
  align-items: center;
}

.error {
  background-color: red;
}

.login {
  background-color: yellow;
  padding: 0.5rem;
  color: darkgray;
}
```

---

true, besides the actual naming, its kind of redudant to include the module inside the module again